### DISCUSSION TITLE
Coming from JoePenna / Dreambooth-Stable-Diffusion

### DISCUSSION THREAD

**Discussion by metzo007** [TheLastBen/fast-stable-diffusion#1285] [General]
Hi,

I started Dreambooth using the JoePenna Dreambooth Repo. I always used the person token, and roundabout 20 pictures of the same person ( 5 of the face, 7 of the upper body and 8 of the whole body), awoiding to have more than 2 with the same backgound or light, and 2000 steps. 

I changed to fast-dreambooth using the same settings. 20 pictures with the same random name and 2000 steps gave me similar results as JoePenna Dreambooth before. 

But now with captions, concept images, 3 different trainings methods with separate learning rate settings and the reduction from 20 to 10 training images as starting point a lot has changed.

I did leave everything by default, and only increased the UNet_Training_Steps from 650 to 1300 to account for using 20 instead of 10 images. I got ok results in overall quality of the picture, but the model is very unresponsive to any prompt. 

Can someone share good settings coming from JoePenna? I thing for training a person captions are not to usefull but concept images might help. 

@TheLastBen. I think a easy-dreambooth with only a photo upload is a good idea. :-)  

Best, metzo

**Comment by TheLastBen** [comment]
Hi, if you want to train on a person, you don't need captions or concept images, keep everything to default, only resume the unet training for 100 steps at a time after testing. the V2 also give better results. make sure you're using a unknown identifier as the images filename without adding (person) or (man/woman)

**Reply by metzo007** [reply]
Does it matter if I start with 10 or 20 pictures?

**Reply by TheLastBen** [reply]
less pictures = faster convergence = lower training steps, start with 10 and 600 steps unet, 200 textenc.

**Reply by metzo007** [reply]
thanks Ben. I always used to manny steps. 10 pictures, 750 steps and 200 textencoder worked best.

**Reply by TheLastBen** [reply]
when you resume training, remember to lower a tiny bit the learning rate

**Reply by metzo007** [reply]
thanks, can you give me a number?

**Reply by TheLastBen** [reply]
8e-6 is fine

**Reply by metzo007** [reply]
That sparks my curiosity. :-) what is the difference between a lower learning rate and lesser steps?

**Reply by TheLastBen** [reply]
let's just say that the learning rate is a more aggressive method for a faster training and also a very low learning rate will make the training useless, the number of steps are effective only when the learning rate is correctly set.

**Reply by ynlissaspill** [reply]
> V2 also give

on dinos the same like 1.5

**Reply by metzo007** [reply]
dinos?
